spec/acceptance/traits_spec.rb

```diff
@@ -20,6 +20,10 @@
           great { "GREAT!!!" }
         end
 
+        trait :great do
```

Comment:
I agree with you that defining a trait twice in the same factory should probably raise an error. Somebody started down that path in https://github.com/thoughtbot/factory_bot/pull/1064. I doesn't seem trivial given the ways traits currently work.